Sources/HummingbirdWebSocket/WebSocketChannel.swift

```diff
@@ -97,13 +96,23 @@ public struct HTTP1WebSocketUpgradeChannel: ServerChildChannel, HTTPChannelHandl
         self.responder = responder
     }
 
+    @available(*, deprecated, renamed: "init(responder:configuration:additionalChannelHandlers:shouldUpgrade:)")
```

Comment:
What's this init needed for?

Reply:
This was the old `init`. Basically I got the arguments in the wrong order, so there was an inconsistency between the two init functions. So I've deprecated one of the old ones and replaced it with a new version with consistent argument order.